### PR TITLE
DOCU-1325 - Success files feature moved into preview section.

### PR DESCRIPTION
**Pull request by lucasdigital**
* Success files feature moved into preview section.
* Link to known issue added to the section.

### CODE REVIEW

File: docs/preview-features.md [hunk 1]
```diff
@@ -69,6 +69,34 @@ Supply the `--source` parameter to the [`filesystem add s3a` command](./command-
 ```text title="Example"
 filesystem add s3a --file-system-id mytarget --bucket-name mybucket1 --credentials-provider org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider --access-key B6ZAI18Z3UIO002Y777A --secret-key OP87Chokisf4hsTP0Q5j95yI904lT7AaDBGJpp0D --source
 ```
+### Configure success files
```
Comment (AshVic) [inline docs/preview-features.md:72]: In general, I was thinking of writing SUCCESS files upper case for consistency?

File: docs/preview-features.md [hunk 2]
```diff
@@ -69,6 +69,34 @@ Supply the `--source` parameter to the [`filesystem add s3a` command](./command-
...
+
+This feature is available for HDFS source filesystems. Use success files to determine when a specific directory has migrated successfully and the data within is ready for an application or job to process on the target side.
```
Comment (AshVic) [inline docs/preview-features.md:74]: Hadoop Distributed File System (HDFS) - write in full then abbreviate in parentheses.

Is it really only a directory or can it be a directory or a file? Perhaps we should only refer to "**paths**" being migrated.

**Review by AshVic — CHANGES_REQUESTED** [review]

File: docs/preview-features.md (hunk 1 above)
Comment (lucasdigital) [reply docs/preview-features.md:72]: Agreed on sticking with small case? The JIRA issue put the upper case version forward, but it's just the value we suggested for customers.

File: docs/preview-features.md (hunk 2 above)
Comment (lucasdigital) [reply docs/preview-features.md:74]: Updated. Absolutely right about the directory, "path" is precisely right. Updated!

**Review by AshVic — APPROVED** [review]

**Pull request merged**